bin/i18n-codeorg/in.sh

```diff
@@ -56,8 +56,12 @@ done
 ### Oceans tutorial
 cp_in apps/node_modules/@code-dot-org/ml-activities/i18n/oceans.json i18n/locales/source/blockly-mooc/fish.json
 
-### AI Lab datasets
+### AI Lab UI Strings
+source_ui=apps/node_modules/@code-dot-org/ml-playground/i18n/mlPlayground.json
+staged_ui=i18n/locales/source/external-sources/ml-playground/mlPlayground.json
```

Comment:
ml-playgorund strings will be located in i18n/locales/source/external-sources/ml-playground in order to keep modularity.